src/main/java/com/twilio/oai/resolver/common/CodegenModelResolver.java

```diff
@@ -45,26 +42,10 @@ public CodegenModel resolve(CodegenModel model, ApiResourceBuilder apiResourceBu
             } else {
                 codegenModelDataTypeResolver.resolve(property, apiResourceBuilder);
             }
-            property.getVendorExtensions().computeIfPresent("x-import", (key, value) -> {
-                if( model.vendorExtensions.containsKey(key)) {
-                    ((HashMap) model.vendorExtensions.get(key)).putAll((Map) value);
-                }
-                else {
-                    model.vendorExtensions.put(key, value);
-                }
-                return value;
-            });
         }
         return model;
     }
     public CodegenModel resolveNestedModel(CodegenProperty property, ApiResourceBuilder apiResourceBuilder) {
```

Comment:
remove this method if not being used

Reply:
Are you referring the resolveNestedModel()?